cic/cic.mk

```diff
@@ -56,6 +56,10 @@ PRODUCT_PACKAGES += \
     Browser2 \
     audio.primary.$(TARGET_PRODUCT)
 
+PRODUCT_PACKAGES += health
```

Comment:
Please check whether this(health) is getting built as part of CIC or even as part of CIV